Fix:
conftest.c:124:33: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))'
https://github.com/oetiker/rrdtool-1.x/pull/1162
--- configure.orig	2022-03-14 09:29:38.000000000 -0500
+++ configure	2022-03-20 16:34:54.000000000 -0500
@@ -19223,6 +19223,7 @@
   gq_cv_have_tm_gmtoff=no
                 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
+#include <stdlib.h>
 #include <time.h>
                                 #include <$ac_cv_struct_tm>
 
@@ -19255,6 +19256,7 @@
   gq_cv_have___tm_gmtoff=no
                 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
+#include <stdlib.h>
 #include <time.h>
                                 #include <$ac_cv_struct_tm>
 
